src/inngest/events/send-slack-message.ts

```diff
@@ -7,35 +8,13 @@ export type SlackMessage = {
   data: SlackMessageEvent
 }
 
-const FieldSchema = z.object({
-  title: z.string().optional(),
-  value: z.string().optional(),
-  short: z.boolean().optional(),
-})
+const Attachments: z.ZodType<MessageAttachment> = z.any()
```

Comment:
web-api had a type for `MessageAttachment` so this replaces my hand crafted type